src/tests/end-to-end/tests/details-view/headings.test.ts

```diff
@@ -73,9 +73,7 @@ describe('Details View -> Assessment -> Headings', () => {
                     headingsPage,
                     detailsViewSelectors.mainContent,
                 );
-                // Note: long-term, length should be 0. The 1 is here as a temporary workaround
-                // for issue https://github.com/dequelabs/axe-core/issues/2459
```

Comment:
This issue was fixed in https://github.com/dequelabs/axe-core/commit/b86c73b7dc61c2609afa6f384502f3c91f9a7f10, included in axe-core 4.0.2